libpages/mime/builder/yarn.lock

```diff
@@ -0,0 +1,71 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Checked in on purpose?

Reply:
Checked in with my consciousness. I think this locks down the node modules versions. Since frontend checks this file in, I figured that might be a good practice.